src/Microsoft.Azure.ServiceBus/Amqp/Framing/AmqpCorrelationFilterCodec.cs

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.Amqp.Framing
+{
+    using System.Text;
+    using Azure.Amqp;
+    using Azure.Amqp.Encoding;
+
+    sealed class AmqpCorrelationFilterCodec : AmqpFilterCodec
+    {
+        public static readonly string Name = AmqpConstants.Vendor + ":correlation-filter:list";
+        public static readonly ulong Code = 0x000001370000009;
+        const int Fields = 9;
+
+        AmqpMap properties;
+
+        public AmqpCorrelationFilterCodec() : base(Name, Code)
+        {
+            this.properties = new AmqpMap();
+        }
+
+        public string CorrelationId { get; set; }
```

Comment:
I have no strong opinion on this. There are a lot of stackOverflow questions with this. For e.g:  [https://stackoverflow.com/questions/2374416/why-use-simple-properties-instead-of-fields-in-c](https://stackoverflow.com/questions/2374416/why-use-simple-properties-instead-of-fields-in-c) 
Leaving this as-is as there is no downside

---
In reply to: [128699407](https://github.com/Azure/azure-service-bus-dotnet/pull/192#discussion_r128699407) [](ancestors = 128699407)